saleor/order/migrations/tasks/saleor3_13.py

```diff
@@ -0,0 +1,31 @@
+from ....celeryconf import app
+from ...models import OrderEvent
+
+# Batch size of size 1000 is about 1MB memory usage in task
+BATCH_SIZE = 1000
+
+
+@app.task
+def order_eventes_rename_transaction_void_events():
+    qs = OrderEvent.objects.filter(type="transaction_void_requested").order_by("pk")
+    ids = qs.values_list("pk", flat=True)[:BATCH_SIZE]
+    if ids:
+        OrderEvent.objects.filter(pk__in=ids).update(
+            type="transaction_cancel_requested"
+        )
+
+        del ids
```

Comment:
Why do you need to `del ids`?

Reply:
I'm afraid of memory leaks here. I believe that celery did tend to have some issues regarding that, so to be fair, I made that preventively